community-tutorials/how-to-install-mongodb-and-mongo-express-with-docker-and-connect-mongo-express-with-mongodb/01-en.md

```diff
@@ -0,0 +1,185 @@
+---
+title: How to install MongoDB and Mongo Express with docker and connect Mongo Express with MongoDB
+description: This tutorial describes how to install a MongoDB database and Mongo Express. Mongo Express is a web based administration tool for the MongoDB database. 
+updated_at: 2021-11-10
+slug: how-to-install-mongodb-and-mongo-express-with-docker-and-connect-mongo-express-with-mongodb
+author_name: Patrick Weber
+author_url: -
+author_image: -
+author_bio: -
+tags: [debian, ubuntu]
+netcup_product_url: https://www.netcup.de/vserver/vps.php
+language: en
+available_languages: en
+---
+
+# Introduction
+This tutorial is about how to install MongoDB and Mongo Express with docker and connect Mongo Express with MongoDB.
+
+# Requirements
+First you need a virtual server with debian or ubuntu from netcup. This tutorial requires also the setup of docker and docker-compose. On the netcup-community-tutorials you find another tutorial how to setup docker and docker-compose on ubuntu.
+
+# Step 1 - First check your docker and docker-compose version
+Check the docker version:
+```
+sudo docker --version
+```
+Output should look like this, maybe with a different version or buildnumber:
+```
+Docker version 20.10.10, build b485636
+```
+
+Check the docker-compose version:
+```
+docker-compose version
+```
+Output should look like this, maybe with another version: 
+```
+Docker Compose version v2.1.0
+```
+
+# Step 4 - Create a docker-compose.yaml file, with MongoDB and Mongo Express
+Create a docker-compose.yaml file with a texteditor:
+```
+vi ./docker-compose.yaml
+```
+
+Insert the following:
+```
+version: '3'
+
+services:
+  mongodb:
+    container_name: mongodb
+    image: mongo:5.0.3
```

Comment:
I had to switch back to version 4.4.6 because my CPU is not supportet by version 5.
```
root@v220211184008168052:~# docker logs mongodb

WARNING: MongoDB 5.0+ requires a CPU with AVX support, and your current system does not appear to have that!
  see https://www.mongodb.com/community/forums/t/mongodb-5-0-cpu-intel-g4650-compatibility/116610/2
  see also https://github.com/docker-library/mongo/issues/485#issuecomment-891991814

/usr/local/bin/docker-entrypoint.sh: line 392:    27 Illegal instruction     (core dumped) "${mongodHackedArgs[@]}" --fork

```

Reply:
ok i changed the version to 4.4.6